### PR TITLE
Milestone 4: Styling and Layout 1.0

### PR DESCRIPTION
**Pull request by abdulrauf11**
The PR contains the first pass at full styling/layout/functionality of the application. It is important that you open the app in a 1920x1080 resolution since it is not responsive.

## Not Included
This PR does not contain the following:
1. Correct sizes - We may have to change the size of a few elements depending on how they look on the final screen.
2. Responsiveness - We may have to make the layout a bit more fluid (if required).
3. Animations - What kind of animations are required for the app (Needs discussion)?
4. Responsive image - The images are taking more time to load so we'll replace them with the gatsby-image component.
5. Correct content - Some of the content (in Arabic) cannot be copied correctly from the Figma design. Also, the modal design on Figma should be updated for accurate results. We've made the modal dynamic, however, we could not copy the content or the correctly sized images for the Figma link.

##  Public Test Link
https://tsck-cochlear-implant.netlify.app/

## Preview
Screenshot 2021-04-18 at 8 23 02 PM

### CODE REVIEW

**Comment by abdulrauf11** [comment]
@tnordberg Is there a way for us to test the application on the final device in the next milestones?

**Comment by tnordberg** [comment]
I am sharing with the SMM team, and we'll get back to you within a couple days. Any changes/feedback that come out of this will likely be applied in the next milestone.

File: src/components/Home/Overview.js [hunk 1]
```diff
@@ -1,21 +1,21 @@
...
-      <p className="formatted">
+      <div className={styles.divider} />
+      <p className={styles.description}>
         <FormattedMessage id="description" />
       </p>
-      <SecodaryInfo />
-    </CardBody>
-  </Card>
+    </div>
+    <SecodaryInfo />
```
Comment (tnordberg) [inline src/components/Home/Overview.js:17]: Small typo: `SecodaryInfo` -> `SecondaryInfo`

**Review by tnordberg — APPROVED** [review]
Nice work. The layout and styling has come a long way already. 
I've written up all the changes and updates required for the next milestone as issues here: https://github.com/scimusmn/tsck-cochlear-implant/issues

Let me know what questions you have, or if you are missing any assets. 

As for animations and transitions, I don't think we require any at this time. If we implement them here, we would need to implement on every interactive in the exhibit, so we are keeping it simple for now.

Thanks!

**Pull request merged**